### ISSUE TITLE
-m gpu Unrecognized

### ISSUE THREAD

**Issue by PoochieMoo** [k4yt3x/video2x#445]
## Component Versions

Please at least fill in the release version and GUI or CLI version.

- **Video2X release version**: 4.8.1
- **CLI version**: 4.3.2

## Symptom

Whenever I run the command line, I get an error telling me that "-m gpu" is an unrecognized argument, and the upscaler won't run.

This is the command line I'm running:
`C:\Users\Me\Downloads\video2x-master\video2x-master\src>python video2x.py -m gpu -d waifu2x_converter_cpp -i "C:\Users\Me\Desktop\Freshman Year College Experience.mp4" -o "C:\Users\Me\Desktop\Freshman Year College Experience_output.mp4"
`

## Error Log or Screenshots

This is the error I get: https://imgur.com/U4WerUH

Basically it says `video2x: error: unrecognized arguments: -m gpu`

**Comment by k4yt3x** [comment]
The way this works has changed a few versions ago. Please check [this example](https://github.com/k4yt3x/video2x#running-video2x-cli). waifu2x-converter-cpp does not have a GPU mode. You can omit that from your command.
